lib/features/app/di/app_scope_register.dart

```diff
@@ -47,6 +47,14 @@ final class AppScopeRegister {
     });
     final logger = LogWriter(surfLogger);
 
+    // TODO(init-project): In order to use Down For Maintenance feature you have to:
+    // - create implementation of [ICheckServerStatusRepository] here
+    // - call [ICheckServerStatusRepository.configure] and await for it
+    // - call [ICheckServerStatusRepository.initImmediateCheck]
+    // - add it to AppScope
+    // - add DownForMaintenanceWidget in builder of MaterialApp in app.dart
+    // Otherwise just delete this.
+
```

Comment:
It is worth putting information about this also in the documentation in the repository itself, and provide link to the documentation here